scripts/opa/main.go

```diff
@@ -0,0 +1,98 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/open-policy-agent/opa/ast"
+	"github.com/open-policy-agent/opa/cmd"
+	"github.com/open-policy-agent/opa/rego"
+	"github.com/open-policy-agent/opa/types"
+
+	parsers "github.com/snyk/snyk-iac-parsers"
+	terraform "github.com/snyk/snyk-iac-parsers/terraform"
+)
+
+func runOpa() {
+	rego.RegisterBuiltin1(
+		&rego.Function{
+			Name:    "hcl2.unmarshal_file",
+			Decl:    types.NewFunction(types.Args(types.S), types.A),
+			Memoize: true,
+		},
+		func(bctx rego.BuiltinContext, a *ast.Term) (*ast.Term, error) {
+
+			var filePath string
+
+			if err := ast.As(a.Value, &filePath); err != nil {
+				return nil, err
+			}
+
+			content, err := ioutil.ReadFile(filePath)
+			if err != nil {
+				return nil, err
+			}
+
+			input, err := terraform.ParseHclToJson("test.tf", string(content), terraform.VariableMap{})
```

Comment:
Yep I pretty much copied this from https://github.com/snyk/cloud-config-opa-policies, and the code there uses `input` and `parsedInput`. I think your suggestion makes a lot of sense sense, but I think I'll leave it as is for consistency and mostly because this code will probably disappear once we update the `snyk-iac-rules` parser.